.github/workflows/ci.yml

```diff
@@ -0,0 +1,98 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+      - "dependabot/**"
+      - "snyk-**"
+  pull_request:
+    branches: [master]
+    types: [opened, reopened]
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
```

Comment:
nit (please feel free to ignore if you deem fit): could we leave a note in the ci.yml file explaining why we chose ubuntu-latest (exactly as you've described in the PR)?

Reply:
good catch, fixed!